### PR TITLE
[TASK] Start the contributor guidelines with the CoC

### PR DESCRIPTION
**Pull request by oliverklee**
This is the first part of #489.

I'd like to add sections bit by bit to make reviewing easier and faster, and to allow us to get things online quickly.

This file will be in sync with its counterpart in the Emogrifier project as much as possible.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Contributing to PHP-CSS-Parser
+
+Those that wish to contribute bug fixes, new features, refactorings and
+clean-up to PHP-CSS-Parser are more than welcome.
+
+When you contribute, please take the following things into account:
+
```
Comment (JakeQZ) [inline .github/CONTRIBUTING.md:7]: Suplerfluous blank line.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
```
Comment (JakeQZ) [inline CODE_OF_CONDUCT.md:33]: Superfluous blank line.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
```
Comment (JakeQZ) [inline CODE_OF_CONDUCT.md:42]: What's 'wiki'?  Is it a legally binding term?

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an
+appointed representative at an online or offline event. Representation of a
+project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at (emogrifier at myintervals dot com).
```
Comment (JakeQZ) [inline CODE_OF_CONDUCT.md:59]: Maybe use a project-specific email address here?

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 1.4, available at
```
Comment (JakeQZ) [inline CODE_OF_CONDUCT.md:73]: Is there a later version?

**Review by JakeQZ — COMMENTED** [review]
Seems mostly fine.  Added some comments.

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (oliverklee) [reply CODE_OF_CONDUCT.md:73]: Yes. I'd like to start with 1.4 to be in sync with what we're using at Emogrifier, and directly after that upgrade both projects to the current version 2.1.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (oliverklee) [reply CODE_OF_CONDUCT.md:42]: Wikipedia is one instance of a wiki. There's also an article on what a wiki is:
https://en.wikipedia.org/wiki/Wiki

GitHub allows enabling a built-in wiki for each repository. You can find the PHP-CSS-Parser wiki in the topmost tab navigation: https://github.com/MyIntervals/PHP-CSS-Parser/wiki

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (oliverklee) [reply CODE_OF_CONDUCT.md:33]: I'd like to keep the CoC as close to the template as much as possible to make upgrades easier. If we think that think need improving there, I suggest we create a PR at their repo instead:
https://github.com/EthicalSource/contributor_covenant/

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (oliverklee) [reply CODE_OF_CONDUCT.md:59]: I'll ping John for this. Until there is such an email address, I'd like to use the existing address.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (oliverklee) [reply .github/CONTRIBUTING.md:7]: Done and repushed.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (JakeQZ) [reply CODE_OF_CONDUCT.md:33]: Seems this comment should be in the general discussion, rather than a reply to specific review comment.

Comment (JakeQZ) [reply CODE_OF_CONDUCT.md:33]: There's still a superfluous blank line.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
There's still one remaining superfluous blank line.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (oliverklee) [reply CODE_OF_CONDUCT.md:33]: Done.

**Review by JakeQZ — APPROVED** [review]
Every journey begins with a single step.  (Or starting the engine.)

**Pull request merged**